packages/domain-picker/src/components/index.tsx

```diff
@@ -144,6 +147,7 @@ const DomainPicker: FunctionComponent< Props > = ( {
 	orderSubDomainsLast = false,
 	onUseYourDomainClick,
 	vendor = getDomainSuggestionsVendor(),
+	hideRecommendationLabel = false,
```

Comment:
In the context of https://github.com/Automattic/wp-calypso/pull/52081/files#r622168030
```suggestion
	showRecommendationLabel = true,
```